### PR TITLE
add a 30 minutes validation buffer

### PR DESCRIPTION
**Pull request by melisabok**
- add a confirmation message when the submission is posted

### CODE REVIEW

**Comment by melisabok** [comment]
@mzarozinski @pmandler I would need help with the wording, what do you think?

screen shot 2017-03-30 at 11 24 53 am

It is a confirmation message when the user submitted the paper.

**Comment by mzarozinski** [comment]
I would remove the "!", and add something like "if you don't see your paper after 15 minutes please contact us"

**Comment by pmandler** [comment]
I agree.

Pam

**Comment by melisabok** [comment]
Great, thanks! I changed it the message.

File: uai2017/webfield/uai2017_webfield.html [hunk 1]
```diff
@@ -303,6 +304,7 @@
 
       view.mkNewNoteEditor(invitation, null, null, user, {
         onNoteCreated: function(idRecord) {
+          promptMessage("Thank you for submitting to UAI. You paper submission will appear shortly in the 'My Submitted Papers' tab. If you don't see your paper after 15 minutes please contact us at info@openreview.net", false, true);
```
Comment (mzarozinski) [inline uai2017/webfield/uai2017_webfield.html]: Should be "Your paper submission", not "You paper submission"

**Comment by melisabok** [comment]
Good catch, fixed it.

**Pull request merged**